app/src/main/java/com/zulip/android/activities/LoginActivity.java

```diff
@@ -260,20 +262,25 @@ public void onTaskFailure(String result) {
             case R.id.legal_button:
                 openLegal();
                 break;
+            case R.id.local_server_button:
+                if (!isInputValid(false)) return;
+                saveServerURL();
+                AsyncDevGetEmails asyncDevGetEmails = new AsyncDevGetEmails(LoginActivity.this);
+                asyncDevGetEmails.execute();
             default:
                 break;
         }
     }
 
-    private boolean isInputValid() {
+    private boolean isInputValid(boolean checkEmailAndPassword) {
```

Comment:
I'll have to create another method or just use this code directly without method ?